src/web/lib/useHover.tsx

```diff
@@ -19,7 +19,7 @@ export function useHover<T>(): [MutableRefObject<T>, boolean] {
                 node.removeEventListener('mouseout', handleMouseOut);
             };
         }
-    }, [ref.current]); // Recall only if ref changes
+    }, []); // Recall only if ref changes
```

Comment:
why this change? Also remove comment

Reply:
Because:

```
React Hook useEffect has an unnecessary dependency: 'ref.current'. Either exclude it or remove the dependency array. Mutable values like 'ref.current' aren't valid dependencies because mutating them doesn't re-render the component.eslint(react-hooks/exhaustive-deps)
```

Thanks for the nudge on the comment 👍